client/src/modules/reports/generate/income_expense/income_expense.html

```diff
@@ -0,0 +1,60 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.INCOME_EXPENSE_REPORT.TITLE</h3>
+      <p class="text-info" translate>REPORT.INCOME_EXPENSE_REPORT.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+          <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate>
+            <bh-fiscal-period-select
+              validation-trigger="ConfigForm.$submitted"
+              formName="ConfigForm"
+              on-select-fiscal-callback="ReportConfigCtrl.onSelectFiscal(fiscal)"
+              on-select-period-from-callback="ReportConfigCtrl.onSelectPeriodFrom(period)"
+              on-select-period-to-callback="ReportConfigCtrl.onSelectPeriodTo(period)">
+            </bh-fiscal-period-select>
+
+            <!-- report type selection  -->
+            <div 
+                class="form-group"
+                ng-class="{ 'has-error' : ConfigForm.$submitted && ConfigForm.type.$invalid }">
+                <label class="control-label">
+                    <i class="fa fa-briefcase"></i> {{ "FORM.SELECT.REPORT_TYPE" | translate }}
```

Comment:
This should use the translate directive and should be changed in the future - this will not block the feature for now.